### PR TITLE
JQuery Lazy Load - Adjustment in type appear

### PR DESCRIPTION
**Pull request by AndersonFriaca**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/tuupola/jquery_lazyload/blob/218e50eb4999fe59ac94b939a65c8c988d1d420b/jquery.lazyload.js#L103>

### CODE REVIEW

File: types/jquery-lazyload/index.d.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ declare namespace JQueryLazyLoad {
         container?: JQuery;
         data_attribute?: string;
         skip_invisible?: boolean;
-        appear?: null;
+        appear?: (elementsLeft: number, options: Options) => void;
```
Comment (mhegazy) [inline types/jquery-lazyload/index.d.ts]: Should `null` still be allowed. the check in https://github.com/tuupola/jquery_lazyload/blob/218e50eb4999fe59ac94b939a65c8c988d1d420b/jquery.lazyload.js#L101, allows `null`.

**Review by mhegazy — CHANGES_REQUESTED** [review]

File: types/jquery-lazyload/index.d.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ declare namespace JQueryLazyLoad {
         container?: JQuery;
         data_attribute?: string;
         skip_invisible?: boolean;
-        appear?: null;
+        appear?: (elementsLeft: number, options: Options) => void | null;
```
Comment (mhegazy) [inline types/jquery-lazyload/index.d.ts]: sorry again. but i do not think this is correct. this is a function that returns `void | null`.
I think what you want is:
```ts
appear?: ((elementsLeft: number, options: Options) => void) | null;
```

**Review by mhegazy — CHANGES_REQUESTED** [review]
Use `((elementsLeft: number, options: Options) => void) | null;` instead of `(elementsLeft: number, options: Options) => void | null`

File: types/jquery-lazyload/index.d.ts (hunk 2 above)
Comment (AndersonFriaca) [reply types/jquery-lazyload/index.d.ts]: Sorry, please check

**Comment by mhegazy** [comment]
thanks!

**Pull request merged**